atat_provisioning_wizard_api.yaml

```
@@ -857,3 +844,13 @@ x-amazon-apigateway-request-validators:
   request-params-validator:
     validateRequestParameters: true
     validateRequestBody: false        
+
+x-amazon-apigateway-gateway-responses:
+  BAD_REQUEST_BODY:
+    statusCode: 400
+    responseTemplates: 
+      application/json: '{ "code": "OTHER", "message": "A valid request body must be provided" }'
+  BAD_REQUEST_PARAMETERS:
```

BAD or MISSING, either is fine.  But think this should include PATH rather than REQUEST.
```suggestion
  MISSING_PATH_PARAMETER:
```

---

I cannot change the name of BAD_REQUEST_PARAMETERS, it is an API Gateway response type and has to have the same name or won't be interpreted correctly with SpecRestApi : https://docs.aws.amazon.com/apigateway/latest/developerguide/supported-gateway-response-types.html
It will catch any missing required parameter, be it PATH or QUERY luckily.